### PR TITLE
Universal EVM blog post changes

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

**Review by bbbeeeee — APPROVED** [review]
Left one comment! I think architecture sections could use a revamp with updated messaging around CAF and eventually PoS but that can be a separate concern.

File: src/pages/about/zetachain.mdx [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 
 ZetaChain's nodes have observers that monitor transactions on every connected
 chain. Through ZetaChain's TSS architecture, the network can sign and verify
-transactions on every connected chain as a wallet can. By being able to read and
-write to connected chains in a secure, decentralized manner, these
-hyper-connected nodes provide a seamless omnichain environment for developers to
-build novel and powerful cross-chain applications on top of.
+transactions on every connected chain. By being able to read and write to
+connected chains in a secure, decentralized manner, these hyper-connected nodes
+provide a seamless omnichain environment for developers to build novel and
```
Comment (bbbeeeee) [inline src/pages/about/zetachain.mdx:33]: a seamless Chain Abstraction Framework for developers to…

File: src/pages/about/zetachain.mdx [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
-blockchain, where all transactions and activity on the platform -- even
-cross-chain transactions -- are fully transparent, verifiable, and function in a
-trust-minimized manner.
+Cosmos SDK and Comet BFT consensus. While many cross-chain solutions like
+bridges often have centralized trust models with track record of exploits and
+hacks, ZetaChain is a Proof-of-Stake blockchain, where all transactions and
+activity on the platform -- even cross-chain transactions -- are fully
+transparent, verifiable, and function in a trust-minimized manner.
 
 ### Hyper-connected nodes
```
Comment (bbbeeeee) [inline src/pages/about/zetachain.mdx:27]: Could we potentially modify this section to change this title to Chain Abstraction Framework?

**Review by bbbeeeee — APPROVED** [review]

File: src/pages/developers/overview.mdx [hunk 3]
```diff
@@ -4,10 +4,10 @@ At a high level, ZetaChain is a Proof of Stake (PoS) blockchain developed using
...
 fast block time and instant finality.
 
-ZetaChain hosts an Ethereum Virtual Machine (EVM) compatible execution layer. In
-addition to supporting all EVM features and standard interactions (such as
-contract creation, contract interaction, and contract composition), ZetaChain's
-EVM distinguishing characteristics include:
+ZetaChain hosts an Ethereum Virtual Machine (EVM) compatible execution layer
+called the Universal EVM. In addition to supporting all EVM features and
+standard interactions (such as contract creation, contract interaction, and
+contract composition), ZetaChain's EVM distinguishing characteristics include:
```
Comment (bbbeeeee) [inline src/pages/developers/overview.mdx:10]: Could we mention Chain Abstraction Framework here?

File: src/pages/index.mdx [hunk 4]
```diff
@@ -2,20 +2,26 @@ import { DocsHighlightBlockRack, SocialButtonsRack } from "~/components/Docs";
 
 # Welcome to ZetaChain docs
 
-Welcome to the official documentation of ZetaChain, the only decentralized blockchain and smart contract platform
-built for omnichain interoperability. ZetaChain supports a truly fluid, multi-chain crypto ecosystem, where users and
-developers can use and appreciate the benefits of any blockchain.
+Welcome to the official documentation of ZetaChain, the only decentralized
```
[the remark this answers is not in the document]
Comment (fadeev) [reply src/pages/index.mdx]: 413e568

File: src/pages/about/zetachain.mdx (hunk 1 above)
Comment (fadeev) [reply src/pages/about/zetachain.mdx]: 7e87ca8

File: src/pages/developers/overview.mdx (hunk 3 above)
Comment (fadeev) [reply src/pages/developers/overview.mdx]: 7730531

**Pull request merged**